modes/calibrate.md

```
@@ -0,0 +1,27 @@
+
+Run the deterministic calibration report and present it. This mode closes the learning loop (#1724): outcomes recorded with `/outcome` are read back and checked against the evaluation scores that preceded them.
+
+> **Non-negotiables:**
+> - **Advisory only.** This mode NEVER edits scoring rules, thresholds, `modes/_shared.md`, or any config. It reports evidence; the user decides what to do with it.
+> - **Deterministic.** The numbers come from `calibrate.mjs` (pure local parsing — no network, no keys, no LLM math). Do not recompute, adjust, or "improve" any rate it prints.
+> - **Honest floors.** If the script withholds a rate as `(n too small)`, present it that way. Never turn a 2-of-3 anecdote into a percentage.
+
+## Pipeline
+
+1. Run the script:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)